homeassistant/components/onvif/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "onvif",
   "name": "ONVIF",
   "documentation": "https://www.home-assistant.io/integrations/onvif",
-  "requirements": ["onvif-zeep-async==0.3.0", "WSDiscovery==2.0.0"],
+  "requirements": ["onvif-zeep-async==0.4.0", "WSDiscovery==2.0.0"],
```

Comment:
It seems https://github.com/hunterjm/python-onvif-zeep-async isn't updated with the released version on pypi. It's nice to be able to look at the source on github to see what changed easily.

Reply:
Sorry, Probably forgot to push. Will do it in a little bit.

Reply:
https://github.com/hunterjm/python-onvif-zeep-async/commit/c457f73169ee50ab6354f478a063e51429b528da